test/sync/src/commonMain/kotlin/io/realm/kotlin/test/mongodb/util/AppAdmin.kt

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2021 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.kotlin.test.mongodb.util
+
+import kotlinx.serialization.json.JsonObject
+
+/**
+ * Wrapper around App Services Server Admin functions needed for tests.
+ */
+interface AppAdmin {
+    val clientAppId: String
```

Comment:
Is this just the `appId`? Calling it `clientAppId` seems to indicate it is something else?

Reply:
This is the name of the field in the server, I think that it is named like that to differentiate it from the `_id` field.